plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -268,15 +276,9 @@ export default class ChatLivePane extends Component {
 
         // Edge case for IOS to avoid blank screens
         // and/or scrolling to bottom losing track of scroll position
-        schedule("afterRender", () => {
-          if (
-            !this._selfDeleted &&
-            !loadingPast &&
-            (this.capabilities.isIOS || !this.isScrolling)
-          ) {
-            this.scrollToMessage(messages[0].id, { position: "end" });
-          }
-        });
+        if (!loadingPast && (this.capabilities.isIOS || !this.isScrolling)) {
```

Comment:
schedule and deleted checks are done in `scrollToMessage` so that's not needed here